disperser/encoder/metrics.go

```diff
@@ -71,6 +73,20 @@ func NewMetrics(httpPort string, logger logging.Logger) *Metrics {
 			},
 			[]string{"size_bucket"},
 		),
+		QueueCapacity: promauto.With(reg).NewGauge(
```

Comment:
This is configured, so doesn't look it should be exported via metric again.

Utilization is a division of queue size (already exported) by capacity.